pdart/pipeline/SuffixInfo.py

```diff
@@ -0,0 +1,420 @@
+        False,
+        "Derived",
+        "Data",
+        "Calibrated {I} image file {F}, corrected for geometric distortion but not CTE, from HST Program {P}",
+        "Calibrated {I} image files, corrected for geometric distortion but not CTE, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "raw"): (  # Why do this?  Possibly remove.
+        True,
+        True,
+        False,
+        "Raw",
+        "Data",
+        "Raw, uncalibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Raw, uncalibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "ima"): (  # Doesn't match handbook.
+        False,
+        False,
+        False,
+        "Partially Processed",
+        "Data",
+        "Calibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Calibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("WFC3", "IR", "flt"): (  # Why do this?  Possibly remove.
+        True,
+        True,
+        False,
+        "Calibrated",
+        "Data",
+        "Calibrated {I} image file {F}, without CR rejection, from HST Program {P}",
+        "Calibrated {I} image files, without CR rejection, from HST Program {P}",
+        ["WFC3"],
+    ),
+    ("NICMOS", "cal"): (
+        True,
+        False,
+        False,
+        "Calibrated",
+        "Data",
+        "Calibrated {I} image file {F} from HST Program {P}",
+        "Calibrated {I} image files from HST Program {P}",
+        ["NICMOS"],
+    ),
+    ("NICMOS", "ima"): (
+        False,
+        False,
+        False,
+        "Partially Processed",
+        "Data",
+        "Calibrated but uncombined {I} image file {F} for MULTIACCUM observations in HST Program {P}",
+        "Calibrated but uncombined {I} image file for MULTIACCUM observations in HST Program {P}",
+        ["NICMOS"],
+    ),
+    ("NICMOS", "mos"): (
+        True,
+        False,
+        False,
+        "Derived",
+        "Data",
+        "Combined, calibrated {I} image file {F} for dithered observations in HST Program {P}",
+        "Combined, calibrated {I} image files for dithered observations in HST Program {P}",
+        ["NICMOS"],
+    ),
+}
+
+# First letter of filenames and the corresponding instrument names
+INSTRUMENTS_INFO = {
+    "i": "wfc3",
+    "j": "acs",
+    "l": "cos",
+    "n": "nicmos",
+    "o": "stis",
+    "u": "wfpc2",
+    "w": "wfpc",
+    "x": "foc",
+    "y": "fos",
+    "z": "ghrs",
+}
+
+# First letter of filenames
+ACCEPTED_INSTRUMENTS = "".join(INSTRUMENTS_INFO.keys()).upper()
+
+# This is used when we only want to download files with shm & spt suffixes.
+PART_OF_ACCEPTED_SUFFIXES = [
+    "SHM",
+    "SPT",
+]
+
+
+# Get the list of suffixes from SUFFIX_INFO based on the boolean values of
+# each key entry. idx = 0 for accepted suffixes, idx = 1 for raw suffixes,
+# idx = 2 for shm suffixes
```

Comment:
Updated SUFFIX_INFO, only one boolean in the dictionary now.
Rename _get_suffixes_list to get_suffixes_list, and it will now take instrument_id as the parameter. If instrument_id is none, get the full list of suffixes from SUFFIX_INFO based on the boolean value at idx=0, else get the list of suffixes for the instrument.